<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(32, 200, 135);
color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: rgb(20, 40, 30);
color:white;
}
</string>
   </property>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="paint">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string>Paint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stopplay">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="Color1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Color 1 </string>
    </property>
   </widget>
   <widget class="QLabel" name="Color2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>580</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Color 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="New">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QSlider" name="Threshold">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="threshold">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="Color11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>540</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Color22">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>580</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Eraser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Eraser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Paint_b">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Paint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_point">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>530</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode point</string>
    </property>
   </widget>
   <widget class="QSlider" name="Thikness">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>580</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="threshold_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Thikness</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
